src/rdkafka_event.c

```diff
@@ -224,6 +224,37 @@ int rd_kafka_event_log (rd_kafka_event_t *rkev, const char **fac,
 	return 0;
 }
 
+int rd_kafka_event_debug_contexts (rd_kafka_event_t *rkev,
+            char *dst, size_t dstsize) {
+        if (unlikely(rkev->rko_evtype != RD_KAFKA_EVENT_LOG))
+                return -1;
+
+        static const char *names[] = {
```

Comment:
variable definitions need to go at the top of the scope, above any code (the if-statement).